pkg/java/pom/parse.go

```diff
@@ -223,7 +223,7 @@ type analysisResult struct {
 	modules              []string
 }
 
-func (p *parser) analyze(pom *pom, exclusions map[string]struct{}) (analysisResult, error) {
+func (p *parser) analyze(pom *pom, exclusions map[string]struct{}, rootDependencyManagement map[string]pomDependency) (analysisResult, error) {
```

Comment:
It doesn't have to be root, right? It just comes from dependents.
- package A
  - package B
    - package C

If I understand correctly, in the above case, package C is affected by `dependencyManagement` in package A and package B. Any other better name? `rootDependencyManagement` sounds only from package A. Or am I wrong?

Reply:
you are right.
But I couldn't think of a better name.

Maybe `rootPomsDependencyManagement` or `rootPackagesDependencyManagement`?
Or perhaps you have some ideas?

Reply:
I came up with `dependentDependencyManagement`, but it could be confusing.